test/write.spec.ts

```diff
@@ -1216,3 +1220,1113 @@ describe('writeBenchmark()', function () {
         }
     });
 });
+
+describe('writeBenchmark() github.enterprise.corp', function () {
```

Comment:
with `describe.each` this second suite will be executed without code duplication :)